libglusterfs/src/inode.c

```diff
@@ -1581,7 +1589,11 @@ inode_table_prune(inode_table_t *table)
                 nlookup = GF_ATOMIC_GET(entry->nlookup);
                 if (nlookup) {
                     if (entry->invalidate_sent) {
-                        list_move_tail(&entry->list, &table->lru);
+                        if (!entry->in_lru_list) {
```

Comment:
If this is false, it's a bug that needs to be fixed. Have you found any case where this happens ?